src/DataCollection.UWP/MainPage.xaml

```diff
@@ -101,19 +42,72 @@
                 MaxWidth="300" />
             <ColumnDefinition Width="*" />
         </Grid.ColumnDefinitions>
-
+        <Grid x:Name="AppTitleBar" Background="Transparent" Grid.ColumnSpan="3">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="Auto" />
+                <ColumnDefinition Width="*" />
+            </Grid.ColumnDefinitions>
+            <Image Source="Assets/Square44x44Logo.png" 
+                   Grid.Column="0" HorizontalAlignment="Left" 
+                   Width="20" Height="20" Margin="5"/>
+        </Grid>
         <!--  Title bar with top buttons  -->
         <CommandBar
-            Grid.Row="0"
+            Grid.Row="1"
             Grid.ColumnSpan="3"
             Background="{x:Bind MainViewModel.ConnectivityMode, Mode=OneWay, Converter={StaticResource ConnectivityModeToColorConverter}}"
             Foreground="{x:Bind MainViewModel.ConnectivityMode, Mode=OneWay, Converter={StaticResource ConnectivityModeToColorConverter}, ConverterParameter=Inverse}">
             <CommandBar.Content>
                 <TextBlock
                     Margin="15,0,0,0"
-                    Style="{StaticResource SubheaderTextBlockStyle}"
+                    Style="{StaticResource SubheaderTextBlockStyle}" FontSize="28"
                     Text="{x:Bind MainViewModel.MapViewModel.Map.Item.Title, Mode=OneWay}" />
             </CommandBar.Content>
+            <AppBarButton
+                x:Uid="ShowBookmarksButton"
```

Comment:
After downloading a map, having a bookmark to return to the view of the offline section would be useful. When downloading a small section, then navigating away with a bookmark, it is hard to find the offline section where a map is visible.